src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ endif(CORECLR_SET_RPATH)
 # Include directories
 
 include_directories(include)
+include_directories(${COREPAL_SOURCE_DIR}/inc/rt)
```

Comment:
This is introducing layering cycle. palrt is meant to be layer above PAL. Can the `_ReturnAdress()` macro be defined in pal.h instead?